native_words.asm

```diff
@@ -598,7 +598,7 @@ accept_total_recall:
                 tya
                 cmp #$80
                 bcc +
-                lda #$7F
+                lda #$7f
```

Comment:
Why a lowercase hex digit?

Reply:
I do agree that they should probably all be the same way.

If you don't have a preference, would you mind making them all uppercase?  I find them much easier to read - especially in the font that Github likes to use for its web interface.